draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -345,13 +345,15 @@ minimum length of the salt, and choice of a hash algorithm.
 
 If the Issuer wants to enable Key Binding, it includes a public key
 associated with the Holder, or a reference thereto.
+The `jwk` member of the `cnf` claim as defined in Section 3.2 of [@!RFC7800] is
+RECOMMENDED for doing so, however, other means MAY be used.
```

Comment:
as Brian said [here](https://github.com/oauth-wg/oauth-selective-disclosure-jwt/issues/389#issuecomment-1858648845)

> This draft intentionally doesn't prescribe how keys are distributed, identified, selected, trusted, etc. any more than JWT/JWS do (e.g. https://datatracker.ietf.org/doc/html/rfc7515#section-6, https://datatracker.ietf.org/doc/html/rfc7515#appendix-D, etc) so I think this is redundant and using `cnf` in examples is sufficient.

```suggestion
```

Reply:
I believe this is a useful area to provide a little more guidance that was previously lacking.

Reply:
The text with that guidance has been changed to not use rfc2119 uppercase key words.